hid-xpadneo/src/hid-xpadneo.c

```diff
@@ -1222,18 +1224,18 @@ static void xpadneo_remove(struct hid_device *hdev)
 
 static const struct hid_device_id xpadneo_devices[] = {
 	/* XBOX ONE S / X */
-	{HID_BLUETOOTH_DEVICE(USB_VENDOR_ID_MICROSOFT, 0x02FD)},
```

Comment:
This is where my indent version acts different to yours. I don't know why, but have seen it before.

Reply:
Version differences?

Reply:
Yeah, maybe... We should probably drop these changes so it doesn't play ping-pong.